src/main/java/com/techiteasy/demo/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,65 @@
+package com.techiteasy.demo.controllers;
+
+import com.techiteasy.demo.exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.*;
+
+//Stap 2: voorzie de klasse van de juiste annotatie
+@RestController
+@RequestMapping
+
+//Deze klasse bevat alle gevraagde CRUD-endpoints
+public class TelevisionsController {
+
+    //Deze mock is een tijdelijke oplossing om data in het geheugen op te slaan, zodat de functionaliteit van de controller getest kan worden zonder een database of externe opslag te gebruiken.
+    private final List<String> televisionDataBase = new ArrayList<>(Arrays.asList("Samsung", "Sony", "LG")); //Ik heb dit 'final' gemaakt om duidelijk te maken dat de referentie naar het Map-object onveranderlijk is na initiele toewijzing (oa voor betere leesbaarheid van de code en het evt. per ongeluk her-toewijzen van het veld elders in de code).
+
+    //Stap 3: Voeg requests toe met ResponseEntity
+    // GET - request voor alle televisies
+    @GetMapping
```

Comment:
Je mappings hebben nog de collectie-naam nodig. (zie de powerpoint slide over restful paden maken) Voor deze mapping is dat dus "/televisions", voor de getById mapping zal dat zijn "/televisions/{id}". Je kunt de collectie-naam ook in de `@`RequestMapping annotatie kwijt. Dan heb je het centraal staan, als je dan toch bananen in plaats van televisies wilt verkopen, hoef je dat maar op 1 plek aan te passen.